src/go/k8s/controllers/redpanda/redpanda_controller.go

```diff
@@ -377,7 +377,7 @@ func (r *RedpandaReconciler) createHelmReleaseFromTemplate(ctx context.Context,
 
 	timeout := rp.Spec.ChartRef.Timeout
 	if timeout == nil {
-		timeout = &metav1.Duration{Duration: 10 * time.Minute}
+		timeout = &metav1.Duration{Duration: 15 * time.Minute}
```

Comment:
This seems ok, but in other commit it is reverted

Reply:
yeah, not sure how that happened, fixing.